config/routes.rb

```diff
@@ -20,4 +21,6 @@
   get '/search/pulmap', to: 'pulmap#show', defaults: { format: 'json' }
   get '/search/staff/', to: 'library_staff#show', defaults: { format: 'json' }
   get '/search/website', to: 'library_website#show', defaults: { format: 'json' }
+
+  mount Flipper::Api.app(Flipper) => '/flipper/api'
```

Comment:
My understanding is that this gives anybody in the world the ability to turn the banner on and off.  I'd much rather limit this to us, either by

- keeping this api but putting it behind some kind of auth,
- requiring us to SSH into a box to access it (via curl, or maybe we just do it via the rails console),
- Maybe a cap task or something if we want to save the trouble of explicitly SSHing into a box,
- something else

Reply:
Yeah. I was trying to find some balance where someone like Beth could turn it on or off, but I think maybe the need for curl / post requests makes that unlikely.

I think maybe the cap task makes the most sense. Maybe we could even put it in an Ansible Tower playbook so you don't have to have your own environment for it.

Reply:
I like the ansible tower idea a lot!  That would be really slick.